pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "beartype>=0.14.0,<0.20.0",
     "hiredis>=3.0.0",
     "litellm>=0.1.1",
-    "aact"
+    "aact @ git+https://github.com/ProKil/aact"
```

Comment:
interesting, why use this one?

Reply:
I think the latest commit with the RestAPINode isn't officially released yet in PyPI